packages/docmanager-extension/src/index.ts

```diff
@@ -783,8 +783,7 @@ function addLabCommands(
   });
 
   commands.addCommand(CommandIDs.rename, {
-    label: () =>
-      trans.__('Rename %1…', fileType(contextMenuWidget(), docManager)),
+    label: () => trans.__('Rename…', fileType(contextMenuWidget(), docManager)),
```

Comment:
It looks like this is missing a %1, which is important (that is the placeholder that is substituted by the file type). Can you put back in that `%1`?

Reply:
Actually, it looks like this line didn't need to be touched?

Reply:
Hi I can put the %1 back in. There is a space after the rename menu item before the ellipsis. You can see the space in the UI. The reason I changed that line was to get rid of the space.